eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/Imp/GovSpec.hs

```diff
@@ -81,6 +83,123 @@ spec =
             voter
             dummyGaid
             [inject $ GovActionsDoNotExist @era $ Set.singleton dummyGaid]
+      it "Motion of no confidence can be passed" $ do
+        modifyPParams $ \pp ->
+          pp
+            & ppDRepVotingThresholdsL . dvtCommitteeNoConfidenceL .~ 1 %! 2
+            & ppPoolVotingThresholdsL . pvtCommitteeNoConfidenceL .~ 1 %! 2
+            & ppCommitteeMaxTermLengthL .~ EpochInterval 200
+        let
+          getCommittee =
+            getsNES $
+              nesEsL . esLStateL . lsUTxOStateL . utxosGovStateL . cgEnactStateL . ensCommitteeL
+          assertNoCommittee =
+            do
+              committee <- getCommittee
+              impAnn "There should not be a committee" $ committee `shouldBe` SNothing
+        assertNoCommittee
+        khCC <- freshKeyHash
+        drep <- setupSingleDRep
+        let committeeMap =
+              Map.fromList
+                [ (khCC, EpochNo 50)
```

Comment:
I think we need to make `electCommittee` accept `Credential`, instead of `KeyHash`, since we will soon have script support. This will also avoid this `mapKeys` below `(Map.mapKeys KeyHashObj committeeMap)`
```suggestion
                [ (KeyHashObj khCC, EpochNo 50)
```